src/exampleTypes.js

```diff
@@ -42,6 +47,11 @@ export default stampKey('type', {
         options: [],
         cardinality: null,
       },
+      label: {
```

Comment:
Do we need these on the defaults? If I'm not mistaken, I think this will add it to the payload of every facet on every saved search because it'll serialize.

The reason we add things to defaults is so we can observe changes to them. Since we don't have any UI that allows changing any of these, I think this whole PR might actually be unnecessary

Reply:
@daedalus28 thanks, i'll check on taking this stuff out

Reply:
@daedalus28 yeah, the whole thing works without this. closing.